libcontainer/rootfs_linux.go

```diff
@@ -81,11 +86,20 @@ func prepareRootfs(pipe io.ReadWriter, iConfig *initConfig, mountFds []int) (err
 		cgroupns:        config.Namespaces.Contains(configs.NEWCGROUP),
 	}
 	for i, m := range config.Mounts {
-		entry := mountEntry{Mount: m}
+		entry := mountEntry{Mount: m, idmapFD: -1}
 		// Just before the loop we checked that if not empty, len(mountFds) == len(config.Mounts).
```

Comment:
This is from #3717, it is not `len(mountFds)`, but `len(mountFds.sourceFds)`. I think line 91-92 could be removed.